Formula/tkey-ssh-agent.rb

```diff
@@ -55,7 +56,15 @@ def caveats
   end
 
   service do
-    run [opt_bin/"tkey-ssh-agent", "--agent-socket", var/"run/tkey-ssh-agent.sock"]
+    args = [
+      opt_bin/"tkey-ssh-agent",
+      "--agent-socket",
+      var/"run/tkey-ssh-agent.sock",
+      "--uss",
+    ]
+    args << "--pinentry" if OS.mac?
+    args << (HOMEBREW_PREFIX/"bin/pinentry-mac") if OS.mac?
+    run args
```

Comment:
It should be done like this: 
https://github.com/Homebrew/homebrew-core/blob/623ee367cf1ed16548bead54ecc47abc2187921f/Formula/podman.rb#L163

Reply:
Ah, I wasn't aware of that syntax. I'll attempt to use that instead.

Reply:
Done.